### ISSUE TITLE
Erroneous print size, SumatraPDF loses its mind.

### ISSUE THREAD

**Issue by Pazns** [sumatrapdfreader/sumatrapdf#221]
When I try to zoom on an image with a "not really suitable" print size declared, like 0 or 3200 centimetres, Sumatra directly goes to 8.33% as farthest zoom (logical), with is way too close to see anything besides pixel soup (annoying).
I can't go back to a proper zoom without choosing "Fit content" or "Fit width" automatic options, and therefore they are the only two usable zooms, since any try to go forward will stuck me again to 8.33% directly.

Is this intended ? Did I miss something ? Any workaround Sumatra-side ?

**Comment by kjk** [comment]
This looks like malformed PDF. The zoom limit is there for a reason - going lower doesn't make sense for properly constructed PDFs.

What do other readers do?

It's possible to be more intelligent but not worth the effort for the rare malformed PDF.

**Comment by Pazns** [comment]
No no, you don't get it.

Sumatra serves also as a standard image viewer, with JPG, PNG, ...
Those can declare a "print size" in various units, like centimetres or pixel density. For some reasons, I got a JPG last day with a print size like "3200cm x 4500cm".
Sumatra uses this print size for its zoom percentages, and therefore it goes nuts with my JPG file.

I looked for a quick way to use the pixel size as reference, at least momentarily.
